cibyl/orchestrator.py

```diff
@@ -200,15 +213,15 @@ def run_query(self, system, start_level=1):
                              key=operator.attrgetter('level'), reverse=True)
         system_methods_stores = defaultdict(SourceMethodsStore)
         last_level = -1
+        if not system.is_enabled():
```

Comment:
Make this the first line of the function. Everything else is useless if it happens.

Reply:
Good point